tardis/widgets/sdec_plot.py

```diff
@@ -867,7 +872,7 @@ def generate_plot_mpl(
         self.ax.set_xlabel(r"Wavelength $[\AA]$", fontsize=12)
         if distance:  # Set y-axis label for flux
             self.ax.set_ylabel(
-                r"$F_{\lambda}$ [erg/s/$\AA/cm^{2}$]", fontsize=12
+                r"$F_{\lambda}$ [erg/s/cm^{2}/$\AA$]", fontsize=12
             )
```

Comment:
good - to remove the italicised units, you could try something along the lines of:
plt.ylabel('Luminosity (erg $\mathrm{s^{-1}}$ $\mathrm{\AA^{-1}}$)') 
This is an example where the labels won't be italicised, since they are in \mathrm{}

Reply:
Ah, sure I can try that for both luminosity and flux labels - but I think I should do this italics fix in a separate PR to adhere to "one PR, one idea" philosophy.

Reply:
sounds good - can you open an issue for it then?

Reply:
it's in #1445

Reply:
nice